pymatgen/io/vasp/outputs.py

```diff
@@ -722,6 +722,10 @@ def run_type(self) -> str:
 
         Hubbard U terms and vdW corrections are detected automatically as well.
         """
+
+        # Care should be taken: if a GGA tag is not specified, VASP will default
+        # to the functional specified by the POTCAR. It is not clear how
+        # VASP handles the "--" value.
```

Comment:
> It is not clear how VASP handles the "--" value.

i think @esoteric-ephemera figured out what happens for `--`

Reply:
It [still runs](https://github.com/materialsproject/custodian/issues/286#issuecomment-1699798494) but it's not a great practice for clarity of results